docs/source/umls/context.puml

```diff
@@ -0,0 +1,44 @@
+@startuml
+!include <C4/C4_Context>
+
+!$light_green = "#43B02A"
+!$dark_green = "#047704"
+
+UpdateElementStyle("person", $bgColor=$dark_green)
+UpdateElementStyle("system", $bgColor=$light_green)
```

Comment:
Hm, my first reaction to all of this was "ew", then "oh cool", and finally after combing through the PlantUML and C4 docs/source code I'm now undecided on how I feel about PlantUML 🤷🏻‍♂️. But I am excited to see what we are able to auto-generate for level 4. Anyhow, I did find that we could tweak the style a little more by modifying the border color but also by using rounded boxes for a (as of 2021) more modern look.
```suggestion
UpdateElementStyle("person", $bgColor=$dark_green, $borderColor=$dark_green, $shape=RoundedBoxShape())
UpdateElementStyle("external_person", $shape=RoundedBoxShape())
UpdateElementStyle("system", $bgColor=$light_green, $borderColor=$light_green, $shape=RoundedBoxShape())
UpdateElementStyle("boundary", $shape=RoundedBoxShape())
```
Which produced this:
<img width="1253" alt="Screen Shot 2021-09-22 at 9 05 12 PM" src="https://user-images.githubusercontent.com/4546435/134444845-b4955a2e-8299-4e6e-a0af-67245063c9e8.png">

Reply:
Yeah, I've done extensive research around which diagram tool we can use that allows us to continue to use Sphinx and generates okay-ish diagrams based on code instead of hosted tools that are harder to embed in the native Conda docs. Using Sphinx is the standard in documenting Python projects, so this is a high priority. I'm not super exciting about having to rely on a Java project FWIW, but alas there isn't a better one IMO. @dbast was nice to send a PR that uses a better bootstrapping process for the jar file at least ;)

Thanks for the style change, totally makes sense. For level 2 and 3 we'll need to move this to a separate file so it's easy to reuse. But I can do that when I'm there.